src/components/ProductListing/Header.jsx

```diff
@@ -0,0 +1,111 @@
+            <li className="search-bar">
+              <form className="input-form flex-column flex-gap-1 flex-grow-1 flex-wrap h-auto w-100">
+                <section className="input-container input-with-icon flex-column b-radius-2 m-5">
+                  <input
+                    id="search"
+                    className="textbox-content p-5"
+                    type="text"
+                    name="search"
+                    placeholder="Search..."
+                    aria-label="Search Products here"
+                  />
+                  <i className="fas fa-search search-icon"></i>
+                </section>
+              </form>
+            </li>
+            <li className="header-nav-icons">
+              <a
+                className="no-link badge p-5 m-5 flex-column justify-content-center align-center"
+                href="https://ripple-store.netlify.app/pages/profile/profile.html"
+              >
+                <span className="badge-icon">
+                  <i className="fas fa-user social"></i>
+                </span>
+              </a>
+            </li>
+            <li className="header-nav-icons h-auto pt-10">
+              <a
+                className="no-link badge p-5 m-5 flex-column justify-content-center align-center"
+                href="https://ripple-store.netlify.app/pages/wishlist/wishlist.html"
+              >
+                <span className="badge-icon">
+                  <i className="fas fa-heart social"></i>
+                </span>
+                <p className="badge-text flex-row justify-content-center align-center top right b-radius-circle">
+                  {/* {cartItemsCount} */}2
+                </p>
+              </a>
+            </li>
+            <li className="header-nav-icons pt-10">
+              <a
+                className="no-link badge p-5 m-5 flex-column justify-content-center align-center"
+                href="https://ripple-store.netlify.app/pages/cart/cart.html"
+              >
+                <span className="badge-icon">
+                  <i className="fas fa-shopping-cart social"></i>
+                </span>
+                <p className="badge-text flex-row justify-content-center align-center top right b-radius-circle">
+                  {/* {cartItemsCount} */}2
+                </p>
+              </a>
+            </li>
+            <li>
+              <i
+                className="fas fa-sun theme-icon social"
+                aria-label="dark/light theme icon"
+              ></i>
+            </li>
+            <li className="header-nav-icons">
+              <a
+                href="https://ripple-store.netlify.app/pages/authentication/authentication.html"
+                aria-label="Login/Register"
+                className="no-link-decoration primary-btn p-3 b-radius-2 my-0 text-bold text-tertiary-color"
+              >
+                Login
+              </a>
+            </li>
```

Comment:
These `li`'s have same structure and are repeating multiple times, So you can extract it out as a separate component

Reply:
ok will do